packages/@aws-cdk/aws-lambda-nodejs/test/integ.dependencies-pnpm.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import { Runtime } from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import * as triggers from '@aws-cdk/triggers';
+import * as lambda from '../lib';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'TestStack');
+
+const handler = new lambda.NodejsFunction(stack, 'Function', {
+  entry: path.join(__dirname, 'integ-handlers/dependencies-pnpm.ts'),
+  runtime: Runtime.NODEJS_18_X,
+  bundling: {
+    minify: true,
+    // Will be installed, not bundled
+    // (axios is a package with sub-dependencies,
+    // will be used to ensure pnpm bundling works as expected)
+    nodeModules: ['axios'],
```

Comment:
Can you add `forceDockerBundling: true`? I think if we force it to bundle inside docker it removes the need to add any of the dependencies in the repo.

Reply:
I switched to Docker bundling, but doing so demonstrated another issue, which is that when you use Docker bundling, the pnpm store is included in the asset by default. This is also the case on `main`.

I've added a tmp directory for the store into the Dockerfile, and added `--config.package-import-method=copy` when running `pnpm install` which resolves this issue